src/main/java/com/splunk/ibm/mq/opentelemetry/Main.java

```diff
@@ -77,6 +82,20 @@ public static void run(
             .registerMetricReader(reader)
             .build();
 
+    // TODO: Prefer to use externally defined config or declarative config, not this weird hybrid
+    System.setProperty("otel.metric.export.interval", "" + config.getTaskDelay());
+    OpenTelemetrySdk otel =
+        AutoConfiguredOpenTelemetrySdk.builder()
+            // TODO: Additional customizations to the sdk as needed here...
+            .build()
+            .getOpenTelemetrySdk();
+
+    Map<String, Meter> meters = new HashMap<>();
+    for (String queueManager : config.getQueueManagerNames()) {
+      meters.put(queueManager, otel.getMeter("opentelemetry.io/mq"));
```

Comment:
is there a reason to do this? we initialize those meters, but do we use somewhere after?

Reply:
Nope, I was going down a path where these could be created in the main and then injected, but ended up going a different route. I'll remove it.